Merge the demo branch into master (#319)
XanaduAI/strawberryfields: doc/tutorials/tutorial_machine_learning.rst

Optimization & machine learning
###############################

> .. warning:: We are currently making some huge changes to the TensorFlow backend of Strawberry Fields, including upcoming TensorFlow 2.0 support. As a result, this tutorial currently is unavailable. Please bare with us as we finish off the work on our new TensorFlow backend!

Author: ```suggestion
.. warning:: We are currently making some huge changes to the TensorFlow backend of Strawberry Fields, including upcoming TensorFlow 2.0 support. As a result, this tutorial currently not working. Please bare with us as we finish off the work on our new TensorFlow backend!
```
